CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Packaging
 
 - Minimum Rust version has been bumped to 1.41.0
+- Minimum Rust version has been bumped to 1.40.0
```

Comment:
This line is redundant now.

Reply:
Yeah, I'll rebase it properly, once I see review on a horizon. The problem is that this patch will require some more work to be included in upstream.